zerver/openapi/zulip.yaml

```diff
@@ -6672,6 +6672,135 @@ paths:
+        [alert words]: /help/pm-mention-alert-notifications#alert-words
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - $ref: "#/components/schemas/SuccessDescription"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      alert_words:
+                        type: array
+                        description: |
+                          An array of strings where each string is an alert word specified by
+                          the requesting user.
+                        items:
+                          type: string
+                    example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "alert_words": ["natural", "illustrious"],
+                      }
+    post:
+      operationId: add-alert-words
+      summary: Add alert words
+      tags: ["users"]
+      description: |
+        Add words to the requesting user's set of [alert words][alert-words].
+
+        [alert-words]: /help/pm-mention-alert-notifications#alert-words
+      parameters:
+        - name: alert_words
+          in: query
+          description: |
+            An array of strings to be added to the requesting user's set of alert words.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+              example: ["foo", "bar"]
+          required: false
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - $ref: "#/components/schemas/SuccessDescription"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      alert_words:
+                        type: array
+                        description: |
+                          An array of strings where each string is an alert word specified by
+                          the requesting user.
+                        items:
+                          type: string
+                    example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "alert_words": ["foo", "bar", "natural", "illustrious"],
+                      }
+    delete:
+      operationId: remove-alert-words
+      summary: Remove alert words
+      tags: ["users"]
+      description: |
+        Remove words from the requesting user's set of [alert words][alert-words].
```

Comment:
I'd update to say, "Remove words (or phrases) from ...", since that's clearly stated in the help center documentation.